### PR TITLE
:memo: store

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

File: packages/k-ramel/doc/STORE.md [hunk 1]
```diff
@@ -16,3 +16,112 @@
...
+> A store holds the whole state tree of your application.\
+> The only way to change the state inside it is to dispatch an action on it.
+
+`k-ramel` store wrap the Redux one. This is convenient to have a simpler `createStore` function, and to add our pre-builds `types` (reducers).\
+With `k-ramel` we want that there is no need to right a custom reducer, and we want to put our logical code into [reactions](./REACTIONS.md), **NOT** into reducers.\
+Also note that the `redux-devtools` is activated by default.
+
+## Example & API
+### Types
+Our types are based on the [k-redux-factyory ones](https://github.com/alakarteio/k-redux-factory/blob/master/TYPES.md):
```
Comment (bpetetot) [inline packages/k-ramel/doc/STORE.md]: typo: *k-redux-factyory ones*

File: packages/k-ramel/doc/STORE.md [hunk 2]
```diff
@@ -16,3 +16,112 @@
...
+ - [subscribe](https://redux.js.org/api/store#subscribe-listener)
+
+and we add some of ours:
+ - drivers
+ - listeners
+    * add
+    * remove
+
+### drivers
+Access the drivers that are plugued into store when you call `createStore`.\
```
Comment (bpetetot) [inline packages/k-ramel/doc/STORE.md]: Access the drivers that are **plugged** into store when you call

File: packages/k-ramel/doc/STORE.md [hunk 3]
```diff
@@ -16,3 +16,112 @@
...
+You can read the full [documentation here](./DRIVERS.md).
+
+### listeners
+listeners.add and listeners.remove are used when you need to hot load listeners (and reactions), or remove them.\
```
Comment (bpetetot) [inline packages/k-ramel/doc/STORE.md]: add code formating for `listeners.add` and `listeners.remove`

**Comment by fabienjuif** [comment]
👍

**Pull request merged**